### DISCUSSION TITLE
Suggested Features

### DISCUSSION THREAD

**Discussion by Nomad965** [Chia-Network/chia-blockchain#4344] [Ideas]
For the GUI, it would be nice having some more functions, like kill process, multi delete plots, stage monitoring
Possibly even a small wizard to plug in some numbers based on hardware setup to push out recommended settings for initial setup.
For the not so tech savvy that would help.
The multinode setup should also be considered, wizard like interface verifying with user if having multiple nodes or standalone on the network, that way it can switch the PNP
A benchmark feature to suggest delay time after 4 or 5 plots to run parallel

**Comment by SackofFluor** [comment]
uff... benchmarking...

I would be happy if "basic" but necessary information would be accessible....

It would just be great, if for example there would be "Buttons" with "Full-Node", "Farmer", "Harvester" which would, for example, have a field where I can add the IP of my harvester/farmer within the local network.

Also, a window where it would show the last attempts and the time it took for the node to verify for proof ( because this is a MAJOR problem for people and I don't even wanna know how many Plots are in reality not farming)

Why can't I access the plotting logs of finished plots... they are stored in the plotter directory anyways....
